src/components/search-main/search-main.js

```diff
@@ -138,10 +138,10 @@ const SearchMain =({
                         <div  className="fw-normal text-muted">
                           <div className="live-region" tabIndex="-1" ref={liveRegionRef}> 
                             {(formSubmitted) && (results.length > 0) &&
-                                <div className="mt-2 px-sm-2 px-md-4 pt-4"><p>Di seguito i migliori risultati per "<strong><mark>{storedInput}</mark></strong>":</p></div>
+                                <div className="mt-2 px-sm-2 px-md-4 pt-4"><p>Questi sono i risultati migliori per "<strong><mark>{storedInput}</mark></strong>":</p></div>
```

Comment:
We can't change this. It support a screen reader user to understand that the results follow "di seguito"...